Sources/Tracking/CustomerIO.swift

```diff
@@ -230,6 +234,17 @@ public class CustomerIO {
             }
         }
     }
+    
+    /**
+     Set the active identifier without calling the customer.io API
+    **/
+    public func setIdentifier(identifier: String,  onComplete: @escaping (Result<Void, CustomerIOError>) -> Void){
```

Comment:
If this function is a synchronous function we don't need to add `onComplete()`

Reply:
did one better. it is a 🔪 function now